docs/tutorials/integrations/kserve/grpc.md

```diff
@@ -84,10 +83,10 @@ kubectl port-forward service/grafana 3000:3000
 
 Open Grafana by going to [http://localhost:3000](http://localhost:3000).
 
-[Add a JSON API data source](http://localhost:3000/connections/datasources/marcusolsson-json-datasource) `Iter8` with the following parameters:
+[Add a JSON API data source](http://localhost:3000/connections/datasources/marcusolsson-json-datasource) `routeguide-test` with the following parameters:
 
 * URL: `http://iter8.default:8080/grpcDashboard` 
-* Query string: `namespace=default&experiment=default`
+* Query string: `namespace=default&experiment=model-test`
```

Comment:
In the `http` tutorial, you have the same data source name as the experiment name. I don't know if you want to do the same here.